File: programmers/2021 Dev-Matching 웹 백엔드 개발자(상반기)/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    ranking = [6, 6, 5, 4, 3, 2, 1]
    zero_cnt = lottos.count(0)
    win_cnt = len([i for i in lottos if i in win_nums])
    answer = [ranking[win_cnt + zero_cnt], ranking[win_cnt]]
    return answer

File: programmers/2021 Dev-Matching 웹 백엔드 개발자(상반기)/로또의 최고 순위와 최저 순위.kt
class Solution {
    fun solution(lottos: IntArray, win_nums: IntArray): IntArray {
        val rank: IntArray = intArrayOf(6, 6, 5, 4, 3, 2, 1)
        var zero_cnt = lottos.filter {it == 0}.size
        var win_cnt = lottos.filter { win_nums.contains(it) }.size
        return intArrayOf(rank[zero_cnt + win_cnt], rank[win_cnt])
    }
}